bazel/utils/BUILD.bazel

```diff
@@ -119,7 +119,7 @@ cc_binary(
 
 #### Actual tests
 remote_run(
-    name = "test-no-machines",
+    name = "test-no-dests",
     rsync_cmd = "./bazel/utils/save-argv test1.rsync",
     rsync_opts = [
         "--test_rsync_opt",
```

Comment:
We should default to `-q` quiet logs too. My stderr from 2 machines was chatty enough, it will get noisier with more hosts.